src/Athena.Core/Models/Room.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Athena.Core.Models
+{
+    public class Room : IUniqueObject<Guid>
```

Comment:
The concept of a model for a `Room` is probably overkill (In fact I reference it in my [proposal](https://github.com/athena-scheduler/design/blob/master/arch/models.md) but never added it which got missed in review). I don't think we need to keep track of all of the associated information. Rooms could probably be just `string` names. The same argument **might** be able to be made for `Campus` objects as well. Thoughts?